src/App.tsx

```diff
@@ -0,0 +1,73 @@
+import React, { Component } from 'react';
+import Auth from './containers/AuthForm/Auth';
+import { Redirect, Route, Switch } from 'react-router';
+import { NavLink } from 'react-router-dom';
+import Button from './components/UI/Button/Button';
+import { Dispatch } from 'redux';
+import { logOut } from './store/auth/actions';
+import { connect } from 'react-redux';
+import { RootState } from './store';
+import { Fragment } from 'react';
+
+interface StateProps {
+  isAuthenticated: boolean
+}
+
+interface DispatchProps {
+  onLogOut: () => void
+}
+
+type Props = StateProps & DispatchProps
+
+class App extends Component<Props> {
+
+  logOutHandler = () => {
+    this.props.onLogOut();
+  };
+
+  render() {
+    // ToDo: Create a navigation component
+
+    const content = () => {
+      if (this.props.isAuthenticated) {
+        return (
+          <Fragment>
```

Comment:
There's a shorthand syntax for fragments:

```jsx
<>this will be in a fragment</>
```

Reply:
That's a great contraction!
Now, I'm using it)